public/tenants/planet/config.ts

```diff
@@ -42,7 +42,7 @@ const config = {
         visible: true,
       },
       me:{
-        title: 'me',
+        title: 'Sign-In',
```

Comment:
Is `Sign-In` changed to something else after the user logged in? Otherwise it can irritate the user and I wouldn't change the menu item name.

Reply:
Maybe we use `Profile` key?